lib/sms.rb

```diff
@@ -0,0 +1,26 @@
+require 'rubygems'
+require 'twilio-ruby'
+
+class SMS
+ 
+ 	def time
+		time = Time.new
+		hour = time.hour + 1
+		min = time.min
+		return "#{hour}: #{min}"
+	end
+
+	 def send_sms
+	# Get your Account Sid and Auth Token from twilio.com/user/account
+	account_sid = 'AC700862d70072900720bd27ca88da1963'
+	auth_token = '78c1646f25e6029d1dbde0a1ccdbe268'
+	@client = Twilio::REST::Client.new account_sid, auth_token
+	 
```

Comment:
Your twilio tokens are now in the public domain on github, and anyone can impersonate you on twilio. Tokens should be placed in your environment variables and never pushed to git. Have a look at the dotenv gem for a simple workaround for this.